doc/HACKING.md

```diff
@@ -26,6 +26,18 @@ Web part of the app is split in three parts: client, worker and common. All code
 directory is intended for GUI and system interaction. Code in the `api`
 contains most of the logic for server communication, encryption, indexing etc.
 
+## Testing
+
+To run a specific test/spec use -f parameter.
```

Comment:
it is already in the "workflow"  section?

Reply:
Missed that and updated the doc, thx!